si2.dal/Entities/ProgramLevel.cs

```diff
@@ -0,0 +1,41 @@
+﻿using si2.dal.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Text;
+using static si2.common.Enums;
+
+namespace si2.dal.Entities
+{
+    [Table("ProgramLevel")]
+    public class ProgramLevel : Si2BaseDataEntity<Guid>, IAuditable
+    {
+        /*
+         * credits : float - mandatory
+         * nameFr : string - mandatory - 30
+         * nameAr : string - mandatory - 30
+         * nameEn : string - mandatory - 30
+         * universityId : Guid - mandatory
+         */
+
+        [Required]
+        public float credits { get; set; }
```

Comment:
properties are capital (CamelCase) Credits
Also decimal is better for Credits
all of them